README.md

```diff
@@ -389,6 +390,52 @@ The secret can be created by executing the following command:
 oc create secret generic keycloak-group-sync --from-literal=username=<username> --from-literal=password=<password>
 ```
 
+### Okta
+
+Groups assigned to Okta applications can be synchronized into OpenShift. The following table describes the set of configuration options for the Okta provider:
+
+| Name | Description | Defaults | Required | 
```

Comment:
Be sure to include all default values

Reply:
One field has a default value that I just added, the others use Go's zero values.  Let me know if you'd like me to specify that (false for extractLoginUsername, for example).

Reply:
If you could specify `false` (it doesnt have to be explicit), but should at least be noted in the README

Reply:
Updated all the fields to have the default value in the table 👍